snomed/com.b2international.snowowl.snomed.core.rest.tests/src/com/b2international/snowowl/snomed/core/rest/ConceptMapCompareDsvExportTest.java

```diff
@@ -153,9 +154,9 @@ private static ConceptMapCompareResultItem createItemUnspecifiedTarget(ConceptMa
 	private static ConceptMapCompareResultItem createItem(ConceptMapCompareChangeKind changeKind, String containerTerm, String sourceId, String sourceTerm, String targetId, String targetTerm) {
 		ConceptMapMapping mapping = ConceptMapMapping.builder()
 				.containerTerm(containerTerm)
-				.sourceComponentURI(ComponentURI.of(CodeSystem.uri("SNOMEDCT-CA"), (short) 100, sourceId))
+				.sourceComponentURI(ComponentURI.of(CodeSystem.uri("SNOMEDCT-CA"), SnomedConcept.TYPE, sourceId))
 				.sourceTerm(sourceTerm)
-				.targetComponentURI(ComponentURI.of(CodeSystem.uri("ICD-10-CA"), (short) 300, targetId))
+				.targetComponentURI(ComponentURI.of(CodeSystem.uri("ICD-10-CA"), SnomedConcept.TYPE, targetId))
```

Comment:
This constant should refer to the ICD-10-CA-specific `concept` value, if there will be one.